src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/LicenceEventType.kt

```diff
@@ -15,6 +15,9 @@ enum class LicenceEventType {
   VARIATION_APPROVED,
   INACTIVE,
   RECALLED,
+  OOS_RECALL,
   VERSION_CREATED,
   NOT_STARTED,
+  OOS_BOTUS,
+  NOT_IN_PILOT,
```

Comment:
these aren't really types of event - they are technically statuses that are not supported
If you're not using them would suggest removing this